.mailmap

```diff
@@ -1,62 +1,176 @@
-Matthias Bussonnier <bussonniermatthias@gmail.com> Matthias BUSSONNIER <bussonniermatthias@gmail.com>
-Michael Droettboom <mdboom@gmail.com> Michael Droettboom <mdroe@stsci.edu>
+
+hannah <story645@gmail.com>
+
+Hans Moritz Günther <moritz.guenther@gmx.de>
+
+J. Goutin <JGoutin@users.noreply.github.com> JGoutin <ginnungagap@free.fr>
+
+Jack Kelly <jack-list@xlk.org.uk> Jack (aka Daniel) Kelly <daniel.kelly10@imperial.ac.uk>
+
+Jaime Fernandez <jaime.frio@gmail.com>
+
+Jake Vanderplas <jakevdp@gmail.com>
+Jake Vanderplas <jakevdp@gmail.com> <jakevdp@yahoo.com>
+Jake Vanderplas <jakevdp@gmail.com> <vanderplas@astro.washington.edu>
+
+James R. Evans <jrevans1@earthlink.net>
+
+Jeff Lutgen <jlutgen@gmail.com> <jlutgen@users.noreply.github.com>
+
+Jeffrey Bingham <bingjeff@gmail.com>
+
+Jens Hedegaard Nielsen <jenshnielsen@gmail.com>
+Jens Hedegaard Nielsen <jenshnielsen@gmail.com> <jens.nielsen@ucl.ac.uk>
+
+John Hunter <jdh2358@gmail.com>
+
+Jorrit Wronski <jowr@mek.dtu.dk>
+
+Joseph Fox-Rabinovitz <jfoxrabinovitz@gmail.com> Mad Physicist <madphysicist@users.noreply.github.com>
+Joseph Fox-Rabinovitz <jfoxrabinovitz@gmail.com> Joseph Fox-Rabinovitz <joseph.r.fox-rabinovitz@nasa.gov>
+
+Jouni K. Seppänen <jks@iki.fi>
+
+Julien Schueller <julien.schueller@gmail.com> <schueller@porsche-l64.phimeca.lan>
+Julien Schueller <julien.schueller@gmail.com> <schueller@bx-l64.phimeca.lan>
+
+Kevin Davies <kdavies4@gmail.com> <daviesk24@yahoo.com>
+
+kikocorreoso <kikocorreoso@gmail.com> <kikocorreoso@users.noreply.github.com>
+
+Klara Gerlei <klarizsofi@gmail.com>
+Klara Gerlei <klarizsofi@gmail.com> klaragerlei <s1466507@sms.ed.ac.uk>
+
+Kristen M. Thyng <kthyng@gmail.com>
+
+Leeonadoh <leo.sunpeng.li@gmail.com>
+
+Lennart Fricke <lennart@die-frickes.eu> <lennart.fricke@kabelmail.de>
+
+Levi Kilcher <levi.kilcher@nrel.gov>
+
+Lion Krischer <krischer@geophysik.uni-muenchen.de>
+
+Martin Fitzpatrick <martin.fitzpatrick@gmail.com> <mfitzp@abl.es>
+
+Matthew Emmett <memmett@gmail.com>
+Matthew Emmett <memmett@gmail.com> <memmett@unc.edu>
+
+Matthias Bussonnier <bussonniermatthias@gmail.com>
+
+matthias <maluethi@protonmail.ch>
+matthias <maluethi@protonmail.ch> <matthias.luethi@lhep.unibe.ch>
+
+Michael Droettboom <mdboom@gmail.com> <mdroe@stsci.edu>
 Michael Droettboom <mdboom@gmail.com> Michael Droettboom <mdboom@debian-vm>
+
+Michiel de Hoon <mjldehoon@yahoo.com>
 Michiel de Hoon <mjldehoon@yahoo.com> Michiel de Hoon <mdehoon@mad002s-MacBook-Air.local>
 Michiel de Hoon <mjldehoon@yahoo.com> Michiel de Hoon <mdehoon@michiel-de-hoons-computer.local>
 Michiel de Hoon <mjldehoon@yahoo.com> Michiel de Hoon <mdehoon@Michiels-MacBook-Pro.local>
 Michiel de Hoon <mjldehoon@yahoo.com> Michiel de Hoon <mdehoon@tkx294.genome.gsc.riken.jp>
+
+MinRK <benjaminrk@gmail.com>
 MinRK <benjaminrk@gmail.com> Min RK <minrk@kerbin.local>
-Nelle Varoquaux <nelle.varoquaux@gmail.com> Varoquaux <nelle.varoquaux@gmail.com>
+
+Moritz Beyreuther <random.seed@web.de>
```

Comment:
Hi @QuLogic, funny to meet you again in this project `:)` (and not in obspy). If you do not mind, I would prefer the pseudonym mbyt as name instead.. I do it like this everywhere on github... Email is right.